src/app/shared/models/dot-workflow/dot-workflow.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface Workflow {
+export interface DotWorkflow {
     id: string;
     name: string;
 }
```

Comment:
The end point return the follow: 

```json
{
id: "85c1515c-c4f3-463c-bac2-860b8fcacc34",
creationDate: 1521045722825,
name: "Default Scheme",
description: "This is the default workflow scheme that will be applied to all content",
archived: false,
mandatory: false,
defaultScheme: true,
modDate: 1521042834952,
entryActionId: null,
},
```

Add the missing attributes to the interface